product_docs/docs/pgd/5/manual/03-configuring-repositories.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: Step 3 - Configuring PGD repositories
+navTitle: Configuring PGD repositories
+deepToC: true
+---
+
+## Configuring PGD repositories
+
+To install and run PGD requires that you configure repositories so that the system can download the appropriate packages.
```

Comment:
```suggestion
To install and run PGD requires that you configure repositories so that the system can download and install the appropriate packages.
```